ui/app/lib/attach-capabilities.js

```diff
@@ -34,7 +34,7 @@ import { isArray } from '@ember/array';
 export default function attachCapabilities(modelClass, capabilities) {
   const capabilityKeys = Object.keys(capabilities);
   const newRelationships = capabilityKeys.reduce((ret, key) => {
-    ret[key] = belongsTo('capabilities', { async: false, inverse: null });
+    ret[key] = belongsTo('capabilities', { async: true, inverse: null });
```

Comment:
just out of curiosity why was it changed to `async: true`?

Reply:
Having it false was causing an error, I can't tell you much more than that!

Reply:
Specifically it caused this test to fail: `auth-config-form options: it submits data correctly`

Reply:
Just looked more into this and I think by updating the test we can leave this as `async: false`. I'll test the options locally as well to make sure they submit properly